app/helpers/classifyIntlMessage.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+export default function (className) {
```

Comment:
can we link to the issue that describes why this is necessary, as in the `notfix` comment?